pkgs/development/libraries/quickfix/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv, fetchFromGitHub, autoconf, automake, libtool }:
+
+let
+  name = "quickfix";
+  version = "1.15.1";
+in stdenv.mkDerivation rec {
+  inherit name;
+
+  nativeBuildInputs = [ autoconf automake libtool ];
+
+  src = fetchFromGitHub {
+    owner = "${name}";
+    repo = "${name}";
+    rev =  "v${version}";
+    sha256 = "1fgpwgvyw992mbiawgza34427aakn5zrik3sjld0i924a9d17qwg";
+  };
+
+  # This ugly mess is to not build or link the unittests, which do not compile with c++17
+  preConfigure = ''
```

Comment:
These substitutions might be better as a patch - replacing partial lines is kind of flaky, and could easily break if upstream changes the files in trivial ways.

Having it as a patch gives a much clearer indication of what the original source was and what the intended change is

Reply:
Redone as an explicit patch, as recommended.